client/my-sites/stats/summary/index.jsx

```diff
@@ -139,11 +182,10 @@ class StatsSummary extends Component {
 		switch ( this.props.context.params.module ) {
 			case 'referrers':
 				title = translate( 'Referrers' );
-				path = 'referrers';
 				statType = 'statsReferrers';
 				summaryView = (
 					<Fragment key="referrers-summary">
-						{ this.renderSummaryHeader( path, statType, false, moduleQuery ) }
+						{ this.renderSummaryHeader( this.getPath( statType ), statType, false, moduleQuery ) }
```

Comment:
It feels we could abstract `this.getPath` to function renderSummaryHeader 🤔

Reply:
I agree with it. The functionality to process the path might be helpful if integrated into the `renderSummaryHeader` function rather than in processing the respective parameter.